packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -291,6 +294,25 @@ ${bodyDetail}`
       if (context.payload.pull_request === undefined) {
         return;
       }
+      if (context.payload.action === 'labeled') {
+        // Only proceeds if `snippet-bot:force-run` label is added.
```

Comment:
non-blocking suggestion for refactor: we can have multiple `app.on(event_list)` handlers. Personally, I like splitting the main logic into a core logic method and have the main event helper deal with the event/payload and call the main logic when necessary. It can make the code easier to follow.

In this case, all this code is at the top (like a guard clause), so it's not too intertwined with the actual business logic.

Reply:
That is a good suggestion. Filed: https://github.com/googleapis/repo-automation-bots/issues/1081